Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -501,7 +506,14 @@ class StatusTableViewController: UITableViewController, UIGestureRecognizerDeleg
                     let numberValue = NSNumber(double: iob).descriptionWithLocale(locale)
                     cell.detailTextLabel?.text = "\(numberValue) Units"
                 } else {
-                    cell.detailTextLabel?.text = emptyValueString
+                    //x22 BIoB Value from IOB Chart Decay Data
+                    dataManager.doseStore.getInsulinOnBoardValues(startDate: NSDate(), endDate: NSDate().dateByAddingTimeInterval(300)) { (values, error) -> Void in
```

Comment:
A tip here: you don't usually want to make asynchronous calls in table data source methods, because you don't own the underlying cell views or control their mapping and reuse. By the time this completion closure executes, the system may have reused the cell for a completely different index path – and your call would be setting label text that's completely out-of-place.

Another issue is that the documentation for `getInsulinOnBoardValues` states that it executes your completion closure on an arbitrary background queue. You cannot use the vast majority of UIKit APIs (like setting label text) on any queue except the main one, so you'll need to create a dispatch call to the main queue. Using UIKit on a background queue can result in unexpected behavior. At best, that means the label text isn't drawn at all until someone else invokes a display call. At worst, the app can crash.

Finally, 
```swift
NSDate().dateByAddingTimeInterval(300)
```
can be more expressively written as
```swift
NSDate(timeIntervalSinceNow: NSTimeInterval(minutes: 5))
```